internal/handlers/handler_register_u2f_step2.go

```diff
@@ -13,7 +13,7 @@ import (
 // to complete the U2F registration.
 func SecondFactorU2FRegister(ctx *middlewares.AutheliaCtx) {
 	responseBody := u2f.RegisterResponse{}
-	err := ctx.ParseBody(&responseBody)
+	_ = ctx.ParseBody(&responseBody)
```

Comment:
@clems4ever, FYI you may want to revert this and actually catch/treat this error.
I don't know the U2F flow very well and do not have the means to be able to test this either.